tasks-by-users/anask601/01-js-portfolio/assets/css/styles.css

```diff
@@ -0,0 +1,1420 @@
+
+  font-size: var(--small-font-size);
+  font-weight: var(--font-medium);
+
+  margin-right: var(--mb-0-25);
+}
+
+.home_scroll_arrow {
+  font-size: 1.25rem;
+}
+/*==================== BUTTONS ====================*/
+.button {
+  display: inline-block;
+
+  padding: 1rem;
+
+  background-color: var(--first-color);
+  color: #fff;
+  font-weight: var(--font-medium);
+
+  border-radius: 0.5rem;
+}
+
+.button:hover {
+  background-color: var(--first-color-alt);
+}
+
+.button_icon {
+  font-size: 1.25rem;
+
+  margin-left: var(--mb-0-50);
+
+  transition: 0.3s;
+}
+
+.button--white {
+  background-color: #fff;
+  color: var(--first-color);
+}
+
+.button--white:hover {
+  background-color: rgb(225, 218, 255);
+}
+
+.button--flex {
+  display: inline-flex;
+  align-items: center;
+}
+
+.button_small {
+  padding: 0.75rem 1rem;
+}
+
+.button_link {
+  padding: 0;
+
+  background-color: transparent;
+
+  color: var(--first-color);
+}
+
+.button_link:hover {
+  background-color: transparent;
+
+  color: var(--first-color-alt);
+}
+/*==================== ABOUT ====================*/
+.about_img {
+  width: 200px;
+
+  border-radius: 0.5rem;
+
+  justify-self: center;
+  align-self: center;
+}
+
+.about_description {
+  text-align: center;
+
+  margin-bottom: var(--mb-2-5);
+}
+
+.about_info {
+  display: flex;
+  justify-content: space-evenly;
+
+  margin-bottom: var(--mb-2-5);
+}
+
+.about_info-title {
+  font-size: var(--h2-font-size);
+  font-weight: var(--font-semi-bold);
+  color: var(--title-color);
+}
+
+.about_info-name {
+  font-size: var(--smaller-font-size);
+}
+
+.about_info-title,
+.about_info-name {
+  display: block;
+  text-align: center;
+}
+
+.about_buttons {
+  display: flex;
+  justify-content: center;
+}
+/*==================== SKILLS ====================*/
+.skills_container {
+  row-gap: 0;
+}
+
+.skills_header {
+  margin-bottom: var(--mb-2-5);
+
+  display: flex;
+  align-items: center;
+
+  cursor: pointer;
+}
+
+.skills_icon,
+.skills_arrow {
+  font-size: 2rem;
+  color: var(--first-color);
+}
+
+.skills_icon {
+  margin-right: var(--mb-0-75);
+}
+
+.skills_titles {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: var(--mb-0-50);
+
+  font-size: var(--h3-font-size);
+}
+
+.skills_subtitle {
+  font-size: var(--small-font-size);
+  color: var(--text-color-light);
+}
+
+.skills_arrow {
+  margin-left: auto;
+  transition: 0.4s;
+}
+
+.skills_list {
+  row-gap: 1.5rem;
+  padding-left: 2.7rem;
+}
+
+.skills_name {
+  font-size: var(--normal-font-size);
+  font-weight: var(--font-medium);
+}
+
+.skills_bar,
+.skills_percentage {
+  height: 5px;
+  border-radius: 0.25rem;
+}
+
+.skills_bar {
+  background-color: var(--first-color-lighter);
+}
+
+.skills_percentage {
+  display: block;
+  background-color: var(--first-color);
+}
+
+.skills_html {
+  width: 90%;
+}
+
+.skills_css {
+  width: 80%;
+}
+
+.skills_js {
+  width: 60%;
+}
+
+.skills_react {
+  width: 85%;
+}
+
+.skills_php {
+  width: 80%;
+}
```

Comment:
Remove this and similar others after making above change

Reply:
Done @AlishaPrasad ma'am